skipper-py/src/bot.py

```diff
@@ -186,19 +186,24 @@ async def run(self):
             # Iterate through each profitable opportunities
             for (transaction, contracts_copy) in transactions_with_contracts:
                 # Build the most profitable bundle from 
-                bidTx: Tx = self.build_most_profitable_bundle(
+                bidTxs: list[Tx] = self.build_most_profitable_bundle(
```

Comment:
J for my understanding, we are creating two bid-txs here to avoid issues in which we aren't able to get into the auction in time? In which case, we are creating a backrun on stale-state...

Reply:
From your usage, has the above been a problem?